### PR TITLE
feat: bump minimum gem versions for opentelemetry-instrumentation-all

### PR DESCRIPTION
**Pull request by ahayworth**
3rd time is the charm, maybe. Bumps minimum gem versions.

### CODE REVIEW

**Comment by arielvalentin** [comment]
@ahayworth I am about to release `que`. May we add it to this?

**Comment by ahayworth** [comment]
Yes, absolutely.

**Comment by arielvalentin** [comment]
Thanks @ahayworth ! I just merged que 0.5.0

File: instrumentation/all/opentelemetry-instrumentation-all.gemspec [hunk 1]
```diff
@@ -42,15 +42,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-instrumentation-koala', '~> 0.19.0'
   spec.add_dependency 'opentelemetry-instrumentation-lmdb', '~> 0.21.0'
   spec.add_dependency 'opentelemetry-instrumentation-mongo', '~> 0.21.0'
-  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.21.0'
+  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.21.1'
   spec.add_dependency 'opentelemetry-instrumentation-net_http', '~> 0.21.0'
-  spec.add_dependency 'opentelemetry-instrumentation-pg', '~> 0.22.0'
+  spec.add_dependency 'opentelemetry-instrumentation-pg', '~> 0.22.1'
   spec.add_dependency 'opentelemetry-instrumentation-que', '~> 0.4.0'
```
Comment (arielvalentin) [inline instrumentation/all/opentelemetry-instrumentation-all.gemspec:48]:
```suggestion
  spec.add_dependency 'opentelemetry-instrumentation-que', '~> 0.5.0'
```

Comment (ahayworth) [reply instrumentation/all/opentelemetry-instrumentation-all.gemspec:48]: Thanks! Once the release actually finishes we'll click this commit button. 😄

**Review by arielvalentin — APPROVED** [review]

**Pull request merged**